geschaeftsordnung.md

```diff
@@ -0,0 +1,111 @@
+# Geschäftsordnung des Fachschaftsrates IT-Systems Engineering
+
+Diese Geschäftsordnung wurde am 04.04.2016 vom Fachschaftsrat beschlossen und tritt in Kraft, sobald die Satzung der Fachschaft IT-Systems Engineering nach einer entsprechenden Änderung die Einführung einer Geschäftsordnung vorsieht.
+
+
+
+##§ 1 Geltungsbereich
+
+Die Geschäftsordnung gilt für den Fachschaftsrat IT-Systems Engineering am Hasso-Plattner-Institut an der Universität Potsdam.
+
+
+##§ 2 Konstituierung
+
+(1) Der Fachschaftsrat tritt spätestens eine Woche nach der Bekanntgabe der Wahlergebnisse auf Einladung des Wahlausschusses zusammen.
+
+(2) Der Fachschaftsrat wählt aus seiner Mitte:
+
+1. einen Vorsitzenden und ggf. dessen Stellvertretenden
+2. einen Finanzreferenten und ggf. dessen Stellvertretenden
+3. einen Beauftragten für die Vernetzung mit den anderen Organen der Studierendenschaft im Sinne der Satzung der Studierendenschaft der Universität Potsdam
+
+(3) Die in (2) festgelegten Ämter können jeweils durch ein konstruktives Misstrauensvotum des Fachschaftsrates oder nach dem Rücktritt einer Person von diesem Amt neu vergeben werden.
+
+
+##§ 3 Aufgabenverteilung
+
+(1) Der Vorsitzende des Fachschaftsrates ist verantwortlich für die Sitzungsleitung gemäß § 5.
+
+(2) Der Finanzreferent verwaltet die Finanzen der Fachschaft gemäß den Beschlüssen des Fachschaftsrates. Er und der ggf. gewählte Stellvertretende sind zu einzeln Verfügungsberechtigten zu erklären. Der Finanzreferent sollte den Finanz-Workshop des Allgemeinen Studierendenausschusses der Universität Potsdam besuchen.
```

Comment:
Das ist normalerweise schon von zu hause möglich, die Überweisungen warten dann halt auf Freigabe im Online Banking. Problem ist, wenn einer warum auch immer ausfällt, gibt halt keine Redundanz. 
Ich sehe durch diese Regelung in der Geschäftsordnung aber kein Problem, da der FSR sie ,auch bevor die Finanzer gewählt werden einfach selbst ändern kann.